src/main/java/com/pinterest/secor/parser/TimestampedMessageParser.java

```diff
@@ -54,11 +54,12 @@ public TimestampedMessageParser(SecorConfig config) {
         LOG.info("FinalizerDelaySeconds: {}", mFinalizerDelaySeconds);
 
         mDtFormatter = new SimpleDateFormat("yyyy-MM-dd");
-        mDtFormatter.setTimeZone(TimeZone.getTimeZone("UTC"));
+        TimeZone timeZone = config.getTimeZone() != null ? TimeZone.getTimeZone(config.getTimeZone()) : TimeZone.getTimeZone("UTC");
```

Comment:
Can you do null or empty string check instead?  And if it's an invalid timezone string, do you want to fall back to UTC?